src/components/forms/ProxyDeviceForm.tsx

```diff
@@ -233,6 +252,13 @@ const ProxyDeviceForm: FC<Props> = ({ formik, project }) => {
         ],
         device.bind,
         "Whether to bind the listen address to the instance or host",
+        (value) => {
```

Comment:
We need to ensure nat mode is not enabled. We can unset it here, in case it is enabled. If it is disabled, we should not update it.